### DISCUSSION TITLE
Datamodel search options

### DISCUSSION THREAD

**Discussion by suresh20041993** [directus/directus#4566] [Feature Requests]
Hi, I am checking data model search options, I could not find them. Please try to enable this.

**Comment by benhaynes** [comment]
I'm not sure what this means, could you clarify the question?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
